Gemfile.lock

```diff
@@ -78,29 +78,29 @@ GEM
       public_suffix (>= 2.0.2, < 5.0)
     airbrussh (1.4.0)
       sshkit (>= 1.6.1, != 1.7.0)
-    annotate (3.0.3)
+    annotate (3.1.1)
       activerecord (>= 3.2, < 7.0)
       rake (>= 10.4, < 14.0)
-    apipie-rails (0.5.16)
+    apipie-rails (0.5.17)
```

Comment:
### [apipie-rails](http://github.com/Apipie/apipie-rails)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.5.16 → 0.5.17

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **7** commits. See the full changes on [the compare page](https://github.com/Apipie/apipie-rails/compare/v0.5.16...v0.5.17).

These are the individual commits:
- (dfac10a) [Allows update metadata for methods (#667)](https://github.com/Apipie/apipie-rails/commit/dfac10a0f1e12874cbbf28f02c1f56f5c96a8fbe)
- (b06d781) [File types are not strings](https://github.com/Apipie/apipie-rails/commit/b06d7817bbcd06e5291cca27c8a100ae903257db)
- (7041806) [Swagger export: Ability to include default values in headers](https://github.com/Apipie/apipie-rails/commit/704180601ce66f7fd198537ceb638fb2701863c6)
- (0a5682c) [exclude rails42 from Ruby 2.5 travis matrix](https://github.com/Apipie/apipie-rails/commit/0a5682cf034344b764cf432df048b2792eb153d3)
- (c768433) [Display engine path in documentation when using api!](https://github.com/Apipie/apipie-rails/commit/c7684334a3b395974be34f7c778e3b9604953a12)
- (103d395) [Update gemfiles with test engine and refactor](https://github.com/Apipie/apipie-rails/commit/103d39538f879e86cb3230da7ec19d71dd933f53)
- (e80c0b0) [Bump version to 0.5.17](https://github.com/Apipie/apipie-rails/commit/e80c0b0c772d370c97a235881b7c2a9fbe2bf987)


</details>